src/main/test/java/IntakeTest.java

```diff
@@ -1,10 +1,6 @@
 import static org.junit.jupiter.api.Assertions.*; 
-
-
 import edu.wpi.first.hal.HAL;
-
 import org.junit.jupiter.api.*;
-
 import frc.robot.subsystems.IntakeSubsystem;
 
 
```

Comment:
You cant move it either.

Reply:
should be fixed now, I will resolve this if the build check passes